jupyterlab/commands.py

```diff
@@ -98,6 +98,14 @@ def ensure_dev(logger=None):
         yarn_proc = Process(['node', YARN_PATH], cwd=parent, logger=logger)
         yarn_proc.wait()
 
+    theme_packages = ['theme-light-extension', 'theme-dark-extension'] 
```

Comment:
Can we use `build:themes` here from the top level?

Reply:
We can, but then they will be built unconditionally, even if they were already built. The other checks in this file only build if the files didn't already exist, so I tried to mirror those.

Reply:
Ah, fair enough.

Reply:
I was considering globbing for all theme packages using [pathlib](https://docs.python.org/3/library/pathlib.html#basic-use), but I was hesitant to differ from the os.path style of the rest of the file.